components/playground/instance/pd.go

```diff
@@ -24,16 +24,32 @@ import (
 	"github.com/pingcap/tiup/pkg/utils"
 )
 
+// PDRole is the role of PD.
+type PDRole string
+
+const (
+	// PDRoleNormal is the default role of PD
+	PDRoleNormal PDRole = "pd"
+	// PDRoleAPI is the role of PD API
+	PDRoleAPI PDRole = "api"
+	// PDRoleTSO is the role of PD TSO
+	PDRoleTSO PDRole = "tso"
+	// PDRoleResourceControl is the role of PD resource control
+	PDRoleResourceControl PDRole = "resource control"
```

Comment:
What about keeping it called `PDRoleResourceManager`?

Reply:
Ok